bank/persistence/persistence_tests/account_related_test.go

```diff
@@ -0,0 +1,64 @@
+package persistance_test
+
+import (
+	"github.com/iliyanmotovski/bankv3/bank/domain"
+	"github.com/iliyanmotovski/bankv3/bank/persistence"
+	"gopkg.in/mgo.v2"
+	"testing"
+	"time"
+)
+
+func TestItWritesReadsUpdatesAndDeletesAccountDataFromDB(t *testing.T) {
```

Comment:
The test looks really huge and tests different behaviours. I suppose it could be split on 2 or maybe 3 tests.